File: Scripts/volatile_and_force_compiler_task/factorial/javascript/factorial.js
var num = 10;

var factorial = function(n) {
    if(n == 0) {
        return 1
    } else {
        return n * factorial(n - 1);
    }
}

var r = 1;

function executeTask(i) {

	return factorial(i);
	
}

for (var i = 0; i < 100000; ++i) { 
	r = executeTask(i);
}

File: scripts/task_tested_with_compile_script/factorial/python/factorial.py
import sys
sys.setrecursionlimit(200000)

def executeTask(i):
	return factorial(i)

def factorial(n):
    z=1
    if n>1:
        z=n*factorial(n-1)
    return z

r = 1
for i in xrange(100000):
	r = executeTask(i)
